play-ahc-ws-standalone/src/main/java/play/libs/ws/ahc/StandaloneAhcWSRequest.java

```diff
@@ -93,24 +91,19 @@ public StandaloneAhcWSRequest setRequestFilter(WSRequestFilter filter) {
         return this;
     }
 
-    /**
-     * Sets a header with the given name, this can be called repeatedly.
-     *
-     * @param name  the header name
-     * @param value the header value
-     * @return the receiving WSRequest, with the new header set.
-     */
     @Override
-    public StandaloneAhcWSRequest setHeader(String name, String value) {
+    public StandaloneAhcWSRequest addHeader(String name, String value) {
         addValueTo(headers, name, value);
         return this;
     }
 
-    /**
-     * Sets a query string
-     *
-     * @param query the query string
-     */
+    @Override
+    public StandaloneAhcWSRequest setHeaders(Map<String, List<String>> headers) {
```

Comment:
I don't get your point. `setHeader` is implemented as a default method at the interface.